pkgs/os-specific/linux/udisks/fix-paths.patch

```diff
@@ -8,109 +8,20 @@ index ca802cce..bfd1c29e 100644
 +SUBSYSTEM=="block", KERNEL=="md*", ENV{DEVTYPE}!="partition", IMPORT{program}="@sh@ -c '@mdadm@ --detail --export $tempnode | @sed@ s/^MD_/UDISKS_MD_/g'"
- 
+
  LABEL="udisks_probe_end"
- 
-diff --git a/modules/zram/data/udisks2-zram-setup@.service.in b/modules/zram/data/udisks2-zram-setup@.service.in
-index ac868e84..03fdd887 100644
---- a/modules/zram/data/udisks2-zram-setup@.service.in
-+++ b/modules/zram/data/udisks2-zram-setup@.service.in
-@@ -8,7 +8,7 @@ Requires=dev-%i.device
- Type=oneshot
- RemainAfterExit=no
- EnvironmentFile=-@zramconfdir@/%i
--ExecStart=-/bin/sh -c 'if [ -n "$ZRAM_NUM_STR" ]; then echo "$ZRAM_NUM_STR" > /sys/class/block/%i/max_comp_streams; fi'
--ExecStart=-/bin/sh -c 'if [ -n "$ZRAM_DEV_SIZE" ]; then echo "$ZRAM_DEV_SIZE" > /sys/class/block/%i/disksize; fi'
--ExecStart=-/bin/sh -c 'if [ "$SWAP" = "y" ]; then mkswap /dev/%i && swapon /dev/%i; fi'
--# ExecStop=-/bin/sh -c 'echo 1 > /sys/class/block/%i/reset'
-+ExecStart=-@sh@ -c 'if [ -n "$ZRAM_NUM_STR" ]; then echo "$ZRAM_NUM_STR" > /sys/class/block/%i/max_comp_streams; fi'
-+ExecStart=-@sh@ -c 'if [ -n "$ZRAM_DEV_SIZE" ]; then echo "$ZRAM_DEV_SIZE" > /sys/class/block/%i/disksize; fi'
-+ExecStart=-@sh@ -c 'if [ "$SWAP" = "y" ]; then @mkswap@ /dev/%i && @swapon@ /dev/%i; fi'
-+# ExecStop=-@sh@ -c 'echo 1 > /sys/class/block/%i/reset'
-diff --git a/modules/zram/udiskslinuxmanagerzram.c b/modules/zram/udiskslinuxmanagerzram.c
-index f647f653..df81e910 100644
---- a/modules/zram/udiskslinuxmanagerzram.c
-+++ b/modules/zram/udiskslinuxmanagerzram.c
-@@ -243,7 +243,7 @@ create_conf_files (guint64   num_devices,
- 
-       g_snprintf (tmp, 255, "zram%" G_GUINT64_FORMAT, i);
-       filename = g_build_filename (PACKAGE_ZRAMCONF_DIR, tmp, NULL);
--      contents = g_strdup_printf ("#!/bin/bash\n"
-+      contents = g_strdup_printf ("#!@bash@\n"
-                                   "# UDisks2 managed ZRAM configuration\n\n"
-                                   "ZRAM_NUM_STR=%" G_GUINT64_FORMAT "\n"
-                                   "ZRAM_DEV_SIZE=%" G_GUINT64_FORMAT "\n"
-diff --git a/src/tests/install-udisks/runtest.sh b/src/tests/install-udisks/runtest.sh
-index e7df4ed2..ab4356d9 100644
---- a/src/tests/install-udisks/runtest.sh
-+++ b/src/tests/install-udisks/runtest.sh
-@@ -1,4 +1,4 @@
--#!/bin/bash
-+#!@bash@
- # vim: dict+=/usr/share/beakerlib/dictionary.vim cpt=.,w,b,u,t,i,k
- # ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
- #
-diff --git a/src/tests/integration-test b/src/tests/integration-test
-index 07e4e029..3bd8ec51 100755
---- a/src/tests/integration-test
-+++ b/src/tests/integration-test
-@@ -299,7 +299,7 @@ class UDisksTestCase(unittest.TestCase):
-         if not device:
-             device = cls.devname(partition)
-         result = {}
--        cmd = subprocess.Popen(['blkid', '-p', '-o', 'udev', device], stdout=subprocess.PIPE)
-+        cmd = subprocess.Popen(['@blkid@', '-p', '-o', 'udev', device], stdout=subprocess.PIPE)
```

Comment:
`blkid` can be removed from the `substituteAll` then.

Reply:
I assume this is being removed because we are not running integration tests. Might be good idea to mention the reason in commit message.

Reply:
Done.